source/GDPR/index.rst

```diff
@@ -13,11 +13,16 @@ GLPI Version Minimum PHP Recommended
 .. note::
    A `basic license <https://services.glpi-network.com/#offers>`__ (or higher) is required. This plugin is also available from the `Cloud <https://glpi-network.cloud/fr/>`__.
 
+
+Install the plugin
+--------------------
+
 -  Go to the marketplace. Download and install the **GDPR Tools** plugin.
 
 .. figure:: images/GDRP-1.png
    :alt:
 
+
```

Comment:
Why adding all those lines?
```suggestion
```

Reply:
I was testing the scale. I'm in the process of going through all the articles again to add a clean-up.